### PR TITLE
Internals action button

### PR DESCRIPTION
**Pull request by kingofkosmos**
Makes internals toggle through clicking the tank's action button, it'll appear like any other action button; when you're holding a tank or have it in any slot. Jetpacks (except inbuilt ones) produce two action buttons, one for internals and one for toggling the jetpack stabilization.

Also changes the status panel to have the internals-stuff stand out a bit more and removes the old toggle internals -button since it's not needed anymore.

Video:
http://webmshare.com/1LdGP

A huge thank you to MrStonedOne and MrPerson for helping me out!

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -148,7 +148,6 @@
 	else
 		R.module.modules += new/obj/item/weapon/tank/jetpack/carbondioxide(R.module)
 		for(var/obj/item/weapon/tank/jetpack/carbondioxide in R.module.modules)
```
Comment (MrStonedOne) [inline code/game/objects/items/robot/robot_upgrades.dm]: You should remove this for loop as well.

File: code/datums/diseases/_MobProcs.dm [hunk 2]
```diff
@@ -127,7 +127,7 @@
 					Cl = M.wear_mask
 					passed = prob((Cl.permeability_coefficient*100) - 1)
 
-	if(!passed && (D.spread_flags & AIRBORNE) && !internals)
+	if(!passed && (D.spread_flags & AIRBORNE))
```
Comment (MrPerson) [inline code/datums/diseases/_MobProcs.dm]: I just want to draw special attention to this fucking line and how messed up it is.

**Comment by Fox-McCloud** [comment]
This is pretty pointless when there's an icon to click on your screen specifically for this--having an action button just adds more clutter.

**Comment by MrPerson** [comment]
That icon is getting removed

**Comment by WJohn** [comment]
This is quite nice, actually, since there's been times when I've wanted to breathe from my pocket oxy tank but actually started breathing from my CO2 jetpack. Speaking of that, does it also show action buttons for breathing out of the jetpack? How do we tell the difference between that and trying to stabilize it?

**Comment by Fox-McCloud** [comment]
Should have looked closer. 

Either case, interesting, it definitely frees up more on the side of the screen.

**Comment by kingofkosmos** [comment]
>Speaking of that, does it also show action buttons for breathing out of the jetpack? How do we tell the difference between that and trying to stabilize it?

Jetpacks produce two action buttons, one looking like the jetpack toggles the stabilization, other looking like an emergency oxygen tank toggles jetpack internals.

**Comment by RemieRichards** [comment]
>no credit for me

I cri.

**Comment by optimumtact** [comment]
Hey nice work, sorry I couldn't help too much with this!

**Comment by kingofkosmos** [comment]
Sry @RemieRichards I forgot, thanks for the help! <3

**Comment by phil235** [comment]
changelog would be nice

**Comment by Shadowlight213** [comment]
Shame, I rather liked the icon, and the amount of screen clutter from action buttons is getting annoying. Ah well.

**Comment by kingofkosmos** [comment]
Do not merge, trying to implement the action button to have a similar look to the old internals toggle -button, showing if the internals are currently on or off.

**Comment by Jalleo** [comment]
How about keeping the internals button where it was as a different action button area?

The way to possibily do it is have the expand button show the options of which tank to use and the one selected is highlighted?

**Comment by kingofkosmos** [comment]
Don't have much time to work on this anymore, plus it exceeds my capabilities by a mile now.

Hopefully someone can make something like this happen. Wjohn even made sprites (http://www.mediafire.com/download/31uwhbwcphj2dbv/Sprites+Request+11.0.5.rar) for every internals tank.

**Comment by bustygoku** [comment]
What needs to happen here is the action button being the gas mask/breath mask rather than the tank. We will lose functionality of adjusting breath masks but that can be changed to alt+click.

**Pull request closed without merging**